schema/table.go

```diff
@@ -207,7 +207,7 @@ func (t *Table) processFields(
 			}
 
 			subtable := newTable(t.dialect, fieldType, seen, canAddr)
-			for _, subfield := range subtable.Fields {
+			for _, subfield := range subtable.allFields {
```

Comment:
`newTable` does not put any fields in `subtable.Fields`, so this for loop is guaranteed to never iterate over something.

It is also the case that on L177 we also iterate over `subtable.allFields`, so maybe this was some kind of copy paste error ?